app/uk/gov/hmrc/nisp/config/wiring/NispCachedStaticHtmlPartialRetriever.scala

```diff
@@ -20,4 +20,12 @@ import com.google.inject.Inject
 import uk.gov.hmrc.http.HttpClient
 import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
 
-class NispCachedStaticHtmlPartialRetriever @Inject()(val httpGet: HttpClient) extends CachedStaticHtmlPartialRetriever
\ No newline at end of file
+import scala.concurrent.duration.Duration
+
+class NispCachedStaticHtmlPartialRetriever @Inject()(val httpGet: HttpClient) extends CachedStaticHtmlPartialRetriever {
+  override def refreshAfter: Duration = ???
```

Comment:
Shouldn't have `NotImplementedExcpetions` in production code, grab me and we can go over what we need to go in here

Reply:
Actually Ste pointed out a better way to do this, grab me